samples/wifi/provisioning/implementation_details.rst

```diff
@@ -0,0 +1,219 @@
+
+========================== ====================================
+Characteristic Name        UUID
+Information                14387801-130c-49e7-b877-2881c89cb258
+Operation Control Point    14387802-130c-49e7-b877-2881c89cb258
+Data Out                   14387803-130c-49e7-b877-2881c89cb258
+========================== ====================================
+
+The characteristic requirements of the Wi-Fi Provisioning Service are shown in the following table.
+
++-----------------+-------------+-------------+-------------+-------------+
+| Characteristic  | Requirement | Mandatory   | Optional    | Security    |
+| Name            |             | Properties  | Properties  | Permissions |
++=================+=============+=============+=============+=============+
+| Information     | Mandatory   | Read        |             | No Security |
+|                 |             |             |             | required    |
++-----------------+-------------+-------------+-------------+-------------+
+| Operation       | Mandatory   | Indicate,   |             | Encryption  |
+| Control         |             | Write       |             | Required    |
+| Point           |             |             |             |             |
++-----------------+-------------+-------------+-------------+-------------+
+| Operation       | Mandatory   | Read, Write |             | Encryption  |
+| Control         |             |             |             | Required    |
+| Point           |             |             |             |             |
+| - Client        |             |             |             |             |
+| Characteristic  |             |             |             |             |
+| Configuration   |             |             |             |             |
+| descriptor      |             |             |             |             |
++-----------------+-------------+-------------+-------------+-------------+
+| Data Out        | Mandatory   | Notify      |             | Encryption  |
+|                 |             |             |             | Required    |
++-----------------+-------------+-------------+-------------+-------------+
+| Data Out        | Mandatory   | Read, Write |             | Encryption  |
+| - Client        |             |             |             | Required    |
+| Characteristic  |             |             |             |             |
+| Configuration   |             |             |             |             |
+| descriptor      |             |             |             |             |
++-----------------+-------------+-------------+-------------+-------------+
+
+The purpose of each characteristic is as follows:
+
+  * `Information`: for client to get `Info` message from server.
+  * `Operation Control Point`: for client to send `Request` message to server, and server to send `Response` message to client.
+  * `Data Out`: for server to send `Result` message to the client.
```

Comment:
response message types can be in lowercase and without other formatting: info, response, and so on.